src/bgw/job.c

```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright (c) 2018  Timescale, Inc. All Rights Reserved.
+ * This file is licensed under the Apache License, see
+ * LICENSE-APACHE at repository root.
```

Comment:
should it say "source root" since repository doesn't really make sense in the context of, e.g., tar-ball distribution.

Reply:
updated.

Reply:
On second thought, I could "source root" being easily confused with the "src" directory.
Other things I've seen are [`top level directory of this distribution`](https://github.com/rust-lang/rust/blob/5ea8eb55cd9f4547b332f43c9f723de30187c223/src/librustc_borrowck/dataflow.rs#L2), [just `LICENSE-APACHE`](https://github.com/rust-lang/rust/blob/5ea8eb55cd9f4547b332f43c9f723de30187c223/src/librustc_borrowck/dataflow.rs#L5), and [no reference to the file](https://github.com/citusdata/citus/blob/431ac805635e291adb5ad5e9605e192e0cebc421/src/backend/distributed/connection/connection_management.c#L1-L9). I'm going to go with `the top level directory of this distribution` for now, let me know if there's something you prefer?

Reply:
FWIW, I think you can skip "of this distribution" and just say "the top-level directory". I think it is implicit that it is the top-level directory of the source code and "distribution" can have other connotations.

Reply:
I think I agree. I was worried that someone would take "top-level directory" to be `/`, but if someone is that confused they're probably hopeless.